src/track/beatgrid.cpp

```diff
@@ -257,13 +249,12 @@ bool BeatGrid::findPrevNextBeats(double dSamples,
     return true;
 }
 
-
 std::unique_ptr<BeatIterator> BeatGrid::findBeats(double startSample, double stopSample) const {
     QMutexLocker locker(&m_mutex);
     if (!isValid() || startSample > stopSample) {
         return std::unique_ptr<BeatIterator>();
     }
-    //qDebug() << "BeatGrid::findBeats startSample" << startSample << "stopSample"
+    // qDebug() << "BeatGrid::findBeats startSample" << startSample << "stopSample"
```

Comment:
We use no space after // for deactivate code.

Reply:
Clang-Format does not distinguish if end-of-line comments contain code or arbitrary text.